wfl/calculators/aims.py

```diff
@@ -65,19 +65,13 @@ def __init__(self, keep_files="default", rundir_prefix="run_Aims_", workdir=None
                  scratchdir=None, calculator_exec=None, **kwargs):
 
         kwargs_command = deepcopy(kwargs)
-        if calculator_exec is not None:
-            if "command" in kwargs:
-                raise ValueError("Cannot specify both calculator_exec and command")
-            if AimsProfile is None:
-                # older syntax
-                kwargs_command["command"] = f"{calculator_exec} > aims.out"
-            else:
-                argv = shlex.split(calculator_exec)
-                try:
-                    kwargs_command["profile"] = AimsProfile(argv=argv)
-                except TypeError:
-                    binary, parallel_info = parse_genericfileio_profile_argv(argv)
-                    kwargs_command["profile"] = AimsProfile(binary=binary, parallel_info=parallel_info)
+
+        if AimsProfile is None:
+            # old syntax
+            warnings.warn("Support for ASE 3.22-style calculator interfase will soon be depreciated")
+            kwargs_command["aims_command"] = calculator_exec
+        elif "profile" not in kwargs_command:
+            kwargs_command["profile"] = construct_aims_profile(calculator_exec, kwargs_command)
```

Comment:
On a second though, I think wfl should expect people to provide `profile` themselves among the kwargs. So I'd leave the two `elif` lines and `construct_aims_profile` out.

Reply:
looks good, except a couple of spelling things: "interface", and "deprecated".

Reply:
Fixed!